### ISSUE TITLE
Add django 1.11 LTS and 2.0 support and remove EOL versions.

### ISSUE THREAD

**Issue by auvipy** [jazzband/django-admin2#460]

**Comment by ChillarAnand** [comment]
With latest version of django, admin2 fails with 

```
 from django.db.models.sql.constants import QUERY_TERMS
ImportError: cannot import name 'QUERY_TERMS' from 'django.db.models.sql.constants' (/Users/curatech/.virtualenvs/library/lib/python3.7/site-packages/django/db/models/sql/constants.py)
```

This seems to be removed in 2.1
https://docs.djangoproject.com/en/2.1/releases/2.1/#features-removed-in-2-1

**Comment by galuszkak** [comment]
This can be closed as 1.11 and 2.0 shouldn't be supported anymore.
